.github/workflows/pull-request.yml

```
@@ -260,6 +260,85 @@ jobs:
+            fi
+          done
+
+          if [ $timeout -eq 0 ]; then
+            echo "Timeout waiting for development application to start"
+            exit 1
+          fi
+          echo "Development app started successfully"
+
+      - name: Stop Development App
+        if: always()
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:8: Quote this to prevent word splitting

(shellcheck)